src/Psy/Command/TimerCommand.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Psy\Command;
+
+use Psy\Configuration;
+use Psy\Shell;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+
+/**
+ * Class TimerCommand
+ * @package Psy\Command
+ */
+class TimerCommand extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('timeit')
+            ->setDefinition(array(
+                new InputArgument('target', InputArgument::REQUIRED, 'A target object or primitive to profile.', null),
+            ))
+            ->setDescription('Profiles with a timer.')
+            ->setHelp(
+                <<<HELP
+Time profiling for functions and commands.
+
+e.g.
+<return>>>> timeit \$closure()</return>
+HELP
+            );
+    }
+
+    protected function runTarget($target)
+    {
+        /** @var Shell $shell */
+        $shell = $this->getApplication();
+        $sh = new \Psy\Shell(new Configuration());
+        $sh->setScopeVariables($shell->getScopeVariables());
+        $sh->addCode($target);
+        $shell->writeReturnValue(eval($sh->flushCode()));
```

Comment:
Do you mind giving me a hint about the proper command for running the code inside the shell? I tried some things and i ended doing the eval directly. I will review the code again to see what i can find, but if you know the exact command would be nice. Thanks